### ISSUE TITLE
[FRIO] Size of font for split post does not fit

### ISSUE THREAD

**Issue by loma-one** [friendica/friendica#13520]
- [x] I have searched open and closed issues for duplicates

### Bug Description
The font size for the "replay to" and "xy has shared the" are different sizes.

In the stable version 

```
p.wall-item-announce, .media .time, .media .shared-time, .media .delivery, .media .location, .media .location a {
  font-size: 11px;
  color: #000000;
}
```
pulled. In the RC it does not work for the reference to a shared post.

### Screenshots

![2023-10-08_13-44](https://github.com/friendica/friendica/assets/44441246/8bc69343-cfd0-4b01-963b-123a7da18731)

### Platform Info

Friendica Version: RC

Friendica Source: git

PHP version: 8.1

SQL version: 10.5

**Comment by MrPetovan** [comment]
Paging @xundeenergie

**Comment by xundeenergie** [comment]
First of all... there is something missing.. 
"in reply to" then the dot and then "shared by..."
There is a username missing after "in reply to"

And then... do you think, the "in reply to <username>" part should be the same size as the "shared by" part?
This could become problematic on usernames which are very long... some users tell their story of life in their usernames.... 

Or should i make the "shared by" Part smaller than the "in reply to" part?

On small screens this are 2 lines... do you think, both lines should have the same fontsize too?

![grafik](https://github.com/friendica/friendica/assets/5894829/1c17fcc6-f070-48ed-86fe-64e015f42a6b)

**Comment by MrPetovan** [comment]
Closed by #13527
